ci/glci/alicloud.py

```diff
@@ -113,6 +115,22 @@ def make_image(self) -> glci.model.OnlineReleaseManifest:
         return dataclasses.replace(
             self.release, published_image_metadata=published_image_set)
 
+    # Share image in a hidden way. The account should apply for whitelist
+    def _share_image(self, region_image_map: {}):
+        for region, image_id in region_image_map.items():
+            self.acs_client.set_region_id(region)
+            logger.info(
+                f"share image ({region}/{image_id}) as a hidden image"
+            )
+            req = ModifyImageSharePermissionRequest.ModifyImageSharePermissionRequest()
+            req.set_ImageId(image_id)
+            # HIDDEN --> share
+            # PRIVATE --> un-share
+            req.set_LaunchPermission('HIDDEN')
```

Comment:
could you:
1. extract the permission values e.g. as an enum
2. pass-in the visibility into this method as a parameter (it is of course OK to assign `HIDDEN` as default value)
?

Reply:
Regarding _deletion_, is any reference function? I need input and output.

Reply:
sure, there is an example for AWS:

https://github.com/gardenlinux/gardenlinux/blob/main/ci/glci/aws.py#L257

Note that this is not yet _used_ (I used the function manually via a REPL shell some times for purging images I published for testing purposes)